proto/spacemesh/types.proto

```diff
@@ -0,0 +1,162 @@
+syntax = "proto3";
+package spacemesh;
+
+message SimpleInt {
+    uint64 value = 1;
+}
+
+message SimpleString {
+    string value = 1;
+}
+
+// a non-negative coin amount
+message Amount {
+    uint64 value = 1;
+}
+
+message AccountId {
+    bytes address = 1;
+}
+
+message Account {
+    AccountId address = 1;  // account public address
+    bytes public_key = 2;   // nice to include the account full public key
+    uint64 counter = 3;     // aka nonce
+    Amount balance = 4;     // known account balance
+}
+
+enum TransactionProcessingStatus {
+    UNDEFINED = 0;
+    REJECTED = 1;               // rejected pre STF processing. e.g. bad data
+    INSUFFICIENT_FUNDS = 2;     // rejected pre STF processing by funds check.
+    CONFLICTING = 3;            // rejected pre STF due to conflicting counter.
+    PENDING = 4;                // pending processing by STF
+    //PROCESSING = 5;
+    //APPROVED = 6;
+    PROCESSED = 7;              // processed. Results are in the tx receipt.
+}
+
+enum TransactionType {
+    COIN = 0;            // simple coin transaction
```

Comment:
s/COIN/SIMPLE/ ?

Reply:
I don't understand this comment - can you please expand a bit?

Reply:
Can we call this a "SIMPLE" transaction rather than a "COIN" transaction?